python/tutorials/06-fused-attention.py

```diff
@@ -200,7 +200,10 @@ class _attention(torch.autograd.Function):
 
     @staticmethod
     def forward(ctx, q, k, v, sm_scale):
-        BLOCK = 128
+        if torch.version.hip is not None:
```

Comment:
Should we create separate file for these tutorials, similar as we did with test_core_amd.py, or should we add this changes here?

Reply:
I agree. We want to soon deprecate test_core_amd.py and just use test_core.py. Not ready quite yet though.